tests/e2e/steps/steps_def.go

```diff
@@ -527,19 +533,25 @@ func (step *Step) restoreTemplate(res Resource, templateFile, crType string) err
 	return nil
 }
 
-func determineMap(crType string) map[string]string {
+func determineMap(crType string) (map[string]string, error) {
 	mapValues := map[string]string{}
 	if crType == "pflex" {
 		mapValues = pflexSecretMap
-	}
-	if crType == "pflexAuth" {
+	} else if crType == "pflexAuth" {
 		mapValues = pflexAuthSecretMap
-	}
-	if crType == "pscale" {
+	} else if crType == "pscale" {
 		mapValues = pscaleSecretMap
+	} else if crType == "pscaleAuth" {
+		mapValues = pscaleAuthSecretMap
+	} else if crType == "authSidecar" {
```

Comment:
can we change
 `crType == "authSidecar` to something more pscale specific like pscaleAuthSideCar?
It'll make it cleaner when we add auth sidecar maps for the other drivers.

Reply:
fixed